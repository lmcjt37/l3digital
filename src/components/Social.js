import React from 'react'

import Link from './Link'

const Social = () => (
    <ul className="icons">
        <li>
            <Link
                target="_blank"
                rel="noopener noreferrer"
                href="https://twitter.com/l3digital_uk"
                className="icon alt fa-twitter"
            >
                <span className="label">Twitter</span>
            </Link>
        </li>
        <li>
            <Link
                target="_blank"
                rel="noopener noreferrer"
                href="https://github.com/l3digitaluk"
                className="icon alt fa-github"
            >
                <span className="label">GitHub</span>
            </Link>
        </li>
        <li>
            <Link
                target="_blank"
                rel="noopener noreferrer"
                href="https://www.linkedin.com/in/l3-digital-uk/"
                className="icon alt fa-linkedin"
            >
                <span className="label">LinkedIn</span>
            </Link>
        </li>
    </ul>
)
export default Social
