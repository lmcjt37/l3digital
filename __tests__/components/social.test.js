import React from 'react'
import renderer from 'react-test-renderer'

import Social from '../../src/components/Social'

describe('Social tests', () => {
    it('check it renders correctly with snapshot', () => {
        const tree = renderer.create(<Social />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
