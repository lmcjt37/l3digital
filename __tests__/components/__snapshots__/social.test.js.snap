// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Social tests check it renders correctly with snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <ul
        class="icons"
      >
        <li>
          <a
            aria-label="Twitter"
            class="icon alt fa-twitter"
            href="https://twitter.com/l3digital_uk"
            rel="noopener noreferrer"
            target="_blank"
          >
            <span
              class="label"
            >
              Twitter
            </span>
          </a>
        </li>
        <li>
          <a
            aria-label="Github"
            class="icon alt fa-github"
            href="https://github.com/l3digitaluk"
            rel="noopener noreferrer"
            target="_blank"
          >
            <span
              class="label"
            >
              GitHub
            </span>
          </a>
        </li>
        <li>
          <a
            aria-label="LinkedIn"
            class="icon alt fa-linkedin"
            href="https://www.linkedin.com/in/l3-digital-uk/"
            rel="noopener noreferrer"
            target="_blank"
          >
            <span
              class="label"
            >
              LinkedIn
            </span>
          </a>
        </li>
      </ul>
    </div>
  </body>,
  "container": <div>
    <ul
      class="icons"
    >
      <li>
        <a
          aria-label="Twitter"
          class="icon alt fa-twitter"
          href="https://twitter.com/l3digital_uk"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            class="label"
          >
            Twitter
          </span>
        </a>
      </li>
      <li>
        <a
          aria-label="Github"
          class="icon alt fa-github"
          href="https://github.com/l3digitaluk"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            class="label"
          >
            GitHub
          </span>
        </a>
      </li>
      <li>
        <a
          aria-label="LinkedIn"
          class="icon alt fa-linkedin"
          href="https://www.linkedin.com/in/l3-digital-uk/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            class="label"
          >
            LinkedIn
          </span>
        </a>
      </li>
    </ul>
  </div>,
  "debug": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
